docker-storage-setup.sh

```diff
@@ -964,6 +946,9 @@ setup_extra_lv_fs() {
     return 1
   fi
   if extra_volume_exists $CONTAINER_ROOT_LV_NAME; then
+    if ! mount_extra_volume $CONTAINER_ROOT_LV_NAME $RESOLVED_MOUNT_DIR_PATH; then
+      Fatal "Failed to mount volume $CONTAINER_ROOT_LV_NAME on $RESOLVED_MOUNT_DIR_PATH"
+    fi
```

Comment:
same change needs to be done in setup_docker_root_lv_fs() as well. That path is used to setup extra volume for docker root lv.